lib/bats-core/test_functions.bash

```diff
@@ -42,6 +72,37 @@ run() {
   IFS=$'\n' lines=($output)
   IFS="$origIFS"
   set "-$origFlags"
+
+  # Show results. Without quotes, multiple lines are glommed together into one
+  if [ -n "$output" ]; then
+    printf "%s\n" "$output"
+  fi
+
+  # Check exit status if requested
+  if [[ "$status" -ne 0 ]]; then
```

Comment:
Again, this would be great for an eventual trace feature but not unconditional prints.